### PR TITLE
Buildozer spec editor

### PR DESCRIPTION
**Pull request by aron-bordin**
Spec editor ok.
[Buildozer configurator](https://github.com/aron-bordin/kivy-designer/issues/32)
[Kivy settings -> list item](https://github.com/aron-bordin/kivy-designer/issues/102)

Screenshots:

App screen:

screenshot from 2015-06-08 12-45-33

Available menus:

screenshot from 2015-06-08 12-45-38

raw spec editor:

screenshot from 2015-06-08 12-45-48

raw spec editor checks if there is any error on .spec:
screenshot from 2015-06-08 12-46-04

List with default values(Android permissions, for example):
screenshot from 2015-06-08 12-50-10

### CODE REVIEW

**Comment by aron-bordin** [comment]
If this PR is ok: https://github.com/kivy/kivy/pull/3451 so I can add the IniLexer on raw buildozer spec editor

File: designer/settings/buildozer_spec_buildozer.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  {
+    "type": "title",
+    "title": "Logs"
+  },
+  {
+    "type": "dict",
+    "title": "Log level",
+    "section": "buildozer",
+    "desc": "Buildozer Log Level",
+    "key": "loglevel",
```
Comment (kived) [inline designer/settings/buildozer_spec_buildozer.json]: This should be `log_level`.

**Comment by kived** [comment]
This is looking pretty good. Note the incorrect buildozer.spec key though.

One real issue - I don't think this should be in a popup. I'd rather this was just another open file tab, like when you open a code file, just using this editor instead of a CodeInput.

Also, this needs to be rebased on the current GSOC_aron_bordin branch before merging.

**Comment by aron-bordin** [comment]
PR Updated. 
* Buildozer spec editor in the tabbed panel.
* spec key fixed
* rebased on my branch
screenshot from 2015-06-24 00-40-54

**Comment by akshayaurora** [comment]
:+1:

**Pull request merged**